benchmarks/result_analyzer.py

```diff
@@ -43,7 +54,6 @@ def run(self):
         "accelerator": pd.Series(dtype="str"),
         "accelerator_model": pd.Series(dtype="str"),
         "xla": pd.Series(dtype="str"),
```

Comment:
"xla_flags" was removed here. Is it intentional or accidental reverting?

Reply:
Thanks for catching that. I have added this line back.